samples/basic/servo_motor/boards/bbc_microbit.overlay

```diff
@@ -1,7 +1,8 @@
 /* SPDX-License-Identifier: Apache-2.0 */
 / {
-	aliases {
-		pwm-servo = &sw_pwm;
+	servo: servo {
```

Comment:
This looks like a use case for `/zephyr,user` rather than a custom node like this.

```devicetree
/ {
        zephyr,user { pwms = < ... >; };
};
```

Then you don't need a binding.

Reply:
I prefer to introduce a binding (added now min/max pulse width there as well). Ideally the sample should have a small driver with a custom servo API, this is how things would probably be done in a real application. What sample does is okay for a quick demo, though.